package.json

```diff
@@ -114,21 +118,24 @@
 		"@skyra/saelem": "^3.0.2",
 		"@types/backoff": "^2.5.1",
 		"@types/diff": "^5.0.0",
+		"@types/google-protobuf": "^3.7.4",
 		"@types/he": "^1.1.1",
 		"@types/i18next-fs-backend": "^1.0.0",
-		"@types/ioredis": "^4.22.2",
+		"@types/ioredis": "^4.22.3",
 		"@types/jest": "^26.0.22",
 		"@types/node": "^14.14.37",
-		"@types/node-fetch": "^2.5.9",
+		"@types/node-fetch": "^2.5.10",
 		"@types/pg": "^7.14.11",
 		"@types/ws": "^7.4.1",
-		"@typescript-eslint/eslint-plugin": "^4.20.0",
-		"@typescript-eslint/parser": "^4.20.0",
+		"@typescript-eslint/eslint-plugin": "^4.21.0",
+		"@typescript-eslint/parser": "^4.21.0",
```

Comment:
Revert lest this is going to cause conflicts down the road, i.e. if not merged before sunday and depfu comes around.